mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
Removing `Move`'s `Cancel` override will probably break it, I'm guessing, due to the `path = NoPath` line.

Using the `keepQueue` flag with `Move` won't work out-of-the-box either, since `Move` doesn't use inner or child activities yet, but queues the other `MovePart`s as next activities. So if you cancel `Move` with the `keepQueue` flag at just the wrong time, it would end up running the `MovePart` anyway.

I have a WIP branch that re-writes `Move` using child activities, but it isn't done yet.